packages/xandr/lib/ad_banner.dart

```diff
@@ -47,6 +47,11 @@ class AdBanner extends StatefulWidget {
               (allowNativeDemand == true && nativeAdBuilder != null),
           'nativeAdBuilder must be set if allowNativeDemand is true',
         ),
+        assert(
```

Comment:
Can you please add a docstring here, referencing the documentation, and maybe quoting the relevant lines?
```suggestion
      //opensdk:auto_refresh_interval or adview.setAutoRefreshInterval(long interval): ....
      // see: https://learn.microsoft.com/en-us/xandr/mobile-sdk/show-banners-on-android
        assert(
```